File: dado.py
# Juego de los dados.

import random

def cube1():
    dado = random.randint(1, 6)
    print('You got: ' + str(dado))

def cube2():
    dado1 = random.randint(1, 6)
    dado2 = random.randint(1, 6)
    print('You got: ' + str(dado1) + ' and ' + str(dado2))

print('¿Desea lanzar uno o dos dados? (1. un dado   2. dos dados): ')
res = input()

while res:
    if res == '1':
        cube1()
    elif res == '2':
        cube2()
    else:
        print('Elija, por favor, una de las altenativas válidas.')

    print('¿Desea lanzar uno o dos dados? (1. un dado   2. dos dados   0. salir): ')
    res = input()

    if res == '0':
        break

print('¡Gracias!')

File: dado.c
// Juego de azar en la que se muestra en pantalla un número aleatorio entre el uno y el seis (incluidos), tal como un dado.

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

void cubeNumber1(void);          // Función que se encargará de obtener el número random.
void cubeNumber2(void);

void main(void)
{
    int res;

    printf("\nEste es el maravilloso juego de los dados\n");
    printf("\n¿Desea tirar uno o dos dados? (1. Un dado   2.dos dados): ");
    scanf("%d", &res);

    while(res)
    {
        switch(res)
        {
            case 1: cubeNumber1();
                    break;
            case 2: cubeNumber2();
                    break;
        }

        printf("\n¿Desea tirar uno o dos dados? (1. un dado   2. dos dados   0. salir): ");
        scanf("%d", &res);
    }

    printf("\n¡Gracias!\n");
}

void cubeNumber1(void)
{
    int cube;

    srand(time(NULL));

    cube = rand() % 7;

    printf("\nYou got: %d\n", cube);
}

void cubeNumber2(void)
{
    int cube1, cube2;

    srand(time(NULL));

    cube1 = rand() % 7;
    cube2 = rand() % 7;

    printf("\nYou got: %d and %d\n", cube1, cube2);
}
